blocks/editable/index.js

```diff
@@ -487,16 +474,12 @@ export default class Editable extends Component {
 	}
 
 	setContent( content ) {
-		if ( ! content ) {
-			content = '';
-		}
-
-		content = renderToString( content );
-		this.editor.setContent( content, { format: 'raw' } );
+		this.editor.setContent( content || '' );
 	}
 
 	getContent() {
-		return nodeListToReact( this.editor.getBody().childNodes || [], createTinyMCEElement );
+		// TODO: Improve performance using childNodes
+		return this.editor.getContent();
```

Comment:
In my head, I've been considering a combination of [`cloneNode`](https://developer.mozilla.org/en-US/docs/Web/API/Node/cloneNode) and various DOM querying / manipulation. Something like:

```js
const body = this.editor.getBody().cloneNode( true );
const boguses = body.querySelectorAll( '[data-mce-bogus]' );
for ( var i = 0; i < boguses.length; i++ ) {
	const bogus = boguses[ i ];
	if ( 'all' !== bogus.getAttribute( 'data-mce-bogus' ) {
		while ( bogus.firstChild ) {
			bogus.parentNode.insertBefore( bogus.firstChild, bogus );
		}
	}

	bogus.parentNode.removeChild( bogus );	
}
const content = body.innerHTML;
```

Not really sure how the performance compares though.

Reply:
Yeah, I think we still want to keep getting the raw content and strip what we strip now. I think it might make sense to, just like we do now, loop through all elements and attributes as we need to:

* Unwrap elements with the `data-mce-bogus` attribute.
* Remove elements and their children with the `data-mce-bogus="all"` attribute.
* Remove attributes that starts with `data-mce-`.

Reply:
I'm not feeling confident with all the TinyMCE's nodes requiring cleanup, I wouldn't mind some help in addressing this

Reply:
I can have a look.

Reply:
Added in 43c2feaf95cd476a4d2ecd390553448f56bd07a0.

Reply:
It's kind of a lot of code, but it's fast.